lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -148,9 +148,12 @@ function collectImageElementInfo() {
  */
 /* istanbul ignore next */
 function determineNaturalSize(url) {
-  return new Promise((resolve, reject) => {
+  // @ts-expect-error nativePromise installed by driver.cacheNatives()
```

Comment:
oof, do we lose *all* typechecking of promise types with this? can we 

```
declare global {
  interface Window {
    __nativePromise: typeof Promise
  }
}
```

or something to make it work?

Reply:
def this, if we don't do what I suggest in my comment above.